crates/nu-std/std/prelude/mod.nu

```
@@ -46,30 +46,3 @@ export def pwd [
         $env.PWD
     }
 }
-
-# Mark command as a test
-export alias "attr test" = echo
-
-# Mark a test command as part of a test suite
-export alias "attr test suite" = echo
-
-# Mark a test command to be ignored
-export alias "attr ignore" = echo
-
-# Mark a command to be run before each test
-export alias "attr before-each" = echo
-
-# Mark a command to be run once before all tests
-export alias "attr before-all" = echo
-
-# Mark a command to be run after each test
-export alias "attr after-each" = echo
-
-# Mark a command to be run once after all tests
-export alias "attr after-all" = echo
-
-# Mark a command as a benchmark
-export alias "attr bench" = echo
-
-# Mark a command as deprecated
-export alias "attr deprecated" = echo
```

wait, where did deprecated go :sweat_smile:

---

maybe deprecated should be an actual built-in command attribute so we can report a parse error? i'm actually not entirely sure how that would work since we need to report the error on call and not on `def`. hmm. still, maybe we keep it in prelude if this is something we think we would want to implement properly later?

---

We can add a `is_deprecated` method to `Command` (or use the command's category), use `@deprecated` to attach the information to custom commands (similar to examples and search terms), and emit warnings on all calls to deprecated commands.

If we go that route `attr deprecated` should be a builtin like the other special attributes (`attr example`, `attr search-terms`)

---

good call. let's hold off on doing anything with this for now i think